CrustTests/Company.swift

```diff
@@ -14,6 +14,35 @@ class Company {
 
 extension Company: AnyMappable { }
 
+enum CompanyCodingKey: Keypath {
+    case uuid
+    case employees(Set<EmployeeCodingKey>)
```

Comment:
I'm trying to think of how we could avoid some of this boilerplate introduced by having nested objects in here. I think it would be better if we were able to define these as a simple String enum or something
```
enum CompanyCodingKey: String {
    case uuid = "data.uuid",
    case founder = "founder",
    ....
}
```

(Since String implements `KeyPath`, right?)

This obviously means we'd need to do something different with nested objects, to which I thought maybe Mappers could be a little more composable like our existing setup in the app. So instead of the CompanyMapping knowing how to map the nested employees, you initialize it with the EmployeeMapping and tell it where to look for that relationship. That way you could constraint all of your enums to just basic properties and maybe clean up some of the syntax boilerplate in this definition.

I dunno, that might not pan out in the long run, I'm just throwing that out as the closest thing I have to an idea for reducing the develop cost of defining this setup. If that doesn't sound viable then this seems reasonable.